azure-pipelines/azure-pipelines-release.yml

```diff
@@ -1,7 +1,7 @@
 ##
 # Azure Pipeline build file for a releasing to pypi
 
-# Copyright (c) 2019, Microsoft Corporation
```

Comment:
Microsoft CELA has requested that we no longer have the year for the Microsoft copyright.  Can you remove the year.

Reply:
Removed